### ISSUE TITLE
[Feature Request]: Prediction interval from regressions

### ISSUE THREAD

**Issue by wenyingyao** [jasp-stats/jasp-issues#2582]
### Description

After estimating a regression model, allow the users to specify a set of values for the input variable, and produce the point prediction, the confidence interval and prediction interval for a given confidence level

### Purpose

This is one of the key use of regression analysis but it is missing from JASP

### Use-case

See above

### Is your feature request related to a problem?

No

### Is your feature request related to a JASP module?

Regression

### Describe the solution you would like

After estimating a regression model, allow the users to specify a set of values for the input variable, and produce the point prediction, the confidence interval and prediction interval for a given confidence level

### Additional context

After estimating a regression model, allow the users to specify a set of values for the input variable, and produce the point prediction, the confidence interval and prediction interval for a given confidence level

**Comment by tomtomme** [comment]
@wenyingyao 
To clarify. You can get those values graphically atm. But you want them presentet numerically in a table, correct?

**Comment by wenyingyao** [comment]
@tomtomme 
That is correct. This is one feature that minitab has, that allows user to specify a set of values for the input variables, and a confidence level, then produce a table of results including the point prediction, confidence interval, and prediction interval. 
I figure this should be very easy to add, but for students who don't know programming or algebra, it is difficult to calculate prediction error by themselves.
Many thanks for the quick response! I'm changing to JASP for my MBA class for the first time, and students really like it. Other than the prediction interval feature, JASP is a perfect substitute of minitab.

**Comment by EJWagenmakers** [comment]
This would be nice to have!

**Comment by tomtomme** [comment]
@wenyingyao 
I switched my classes to jasp in 2020, so I know the feeling. There will always be something missing :D - but thats okay since there are so many things to enjoy already compared to SPSS. 
Lets hope your request will be implemented soon, but as you can see, we have around 400 valid feature requests at the moment + some bugs to fix :D
Really cool to see how many ideas people have to improve jasp :)
So thx for this one!

**Comment by RemiDav** [comment]
That feature would be applicable to all the models in Regression.
It is the main thing keeping me to use JASP in my class to teach forecasting.
